python_modules/dagster/dagster/_core/workspace/context.py

```diff
@@ -600,57 +594,57 @@ def _load_location(self, origin: RepositoryLocationOrigin) -> WorkspaceLocationE
                 )
             )
 
-        return WorkspaceLocationEntry(
+        return WorkspaceCodeLocationEntry(
             origin=origin,
-            repository_location=location,
+            code_location=location,
             load_error=error,
-            load_status=WorkspaceLocationLoadStatus.LOADED,
+            load_status=WorkspaceCodeLocationLoadStatus.LOADED,
```

Comment:
While we are doing sweeping naming changes we should take out "Workspace" too IMO - anywhere that says "WorkspaceCodeLocation" should just be "CodeLocation"

Reply:
Sounds good-- any other older conventions you can think of that should be changed? I don't have full context on the meaning/history of all these names.